<launch>
  <!--<arg name="manager" />--> <!-- Must be globally qualified -->
  <node pkg="nodelet" type="nodelet" name="stereo_nodelet_manager"  args="manager" output="screen"/>

  <param name="vocabulary_path" value="$(find gf_orb_slam2)/../ORB_Data/ORBvoc.bin" />
  <param name="camera_setting_path" value="$(find gf_orb_slam2)/Examples/ROS/GF_ORB_SLAM2/launch/Spiri_lowRes_Stereo.yaml" />
  <param name="bUseViewer" value="false" />
  <param name="bEnablePublishROSTopic" value="true" />
  <param name="do_rectify" value="false" />
  <param name="budget_per_frame" value="1600" />
  <param name="do_viz" value="false"/>
  <param name="topic_img_l" value="/stereo/left/image_rect"/>
  <param name="topic_img_r" value="/stereo/right/image_rect"/>
  <param name="path_to_traj" value="$(find gf_orb_slam2)/../ORB_Data/log/"/>
  <param name="path_to_map" value="$(find gf_orb_slam2)/../ORB_Data/log/"/>

  <node pkg="nodelet" type="nodelet" name="stereo_nodelet" args="load ORB_SLAM2/StereoNodelet stereo_nodelet_manager " output="screen" >
  </node>
</launch>
